moduleframework/helpers/openshift_helper.py

```diff
@@ -66,7 +66,7 @@ def _app_exists(self):
         if int(oc_status.exit_status) == 0:
             common.print_info("Application already exists.")
             return True
-        oc_services = self.runHost("oc get services -o json", ignore_status=True).stdout
+        oc_services = self.runHost("oc get pods -o json", ignore_status=True).stdout
```

Comment:
this seem odd, are you sure you know what you are doing?

Reply:
I will clarify it. `pods` make sense, but what if application failed. And the other use cases.

Reply:
Based on discussion with OpenShift folks, `oc get pods` is better for getting status if application is READY.